test/plugins/scenarios/go-redisv9/config/excepted.yml

```diff
@@ -18,6 +18,82 @@ segmentItems:
   - serviceName: go-redisv9
     segmentSize: ge 1
     segments:
+      - segmentId: not null
+        spans:
+          - operationName: redis/ping
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:6379
+            skipAnalysis: false
+            tags:
+              - { key: cache.type, value: redis }
+              - { key: cache.op, value: '' }
+              - { key: cache.cmd, value: ping }
+              - { key: cache.key, value: '' }
+              - { key: cache.args, value: not null }
+      - segmentId: not null
+        spans:
+          - operationName: redis/command
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:7001
+            skipAnalysis: false
+            tags:
+              - { key: cache.type, value: redis }
+              - { key: cache.op, value: '' }
+              - { key: cache.cmd, value: command }
+              - { key: cache.key, value: '' }
+              - { key: cache.args, value: 'command: map[]' }
+      - segmentId: not null
+        spans:
+          - operationName: redis/cluster/slots
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:7001
+            skipAnalysis: false
+            tags:
+              - { key: cache.type, value: redis }
+              - { key: cache.op, value: '' }
+              - { key: cache.cmd, value: cluster slots }
+              - { key: cache.key, value: slots }
+              - { key: cache.args, value: 'cluster slots: []' }
+      - segmentId: not null
+        spans:
+          - operationName: redis/ping
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: not null
```

Comment:
Could you please provide the peer address?

Reply:
The image `bitnami/redis-cluster` provides environment variables `REDIS_CLUSTER_ANNOUNCE_IP `, but after trying many times, I couldn't successfully initialize the cluster redis with the specified IP. Also, the Go service couldn't access redis.

Reply:
My intention is not the IP address, after testing it should be the address filled in the program. Isn't it?